packages/odyssey-react-mui/src/labs/SideNav/SortableList/SortableList.tsx

```diff
@@ -99,7 +100,8 @@ export const SortableList = <T extends BaseItem>({
         if (over && active.id !== over?.id) {
           const activeIndex = items.findIndex(({ id }) => id === active.id);
           const overIndex = items.findIndex(({ id }) => id === over.id);
-          onChange(parentId, activeIndex, overIndex);
+          if (items[overIndex].isSortable !== false)
```

Comment:
@ganeshsomasundaram-okta little nit-picky but, don't think we need the `!== false` here right?
```suggestion
          if (items[overIndex].isSortable)
```

Reply:
I made it explicit to disable sorting by passing a false value for an item within a sortable list. If it is not passed in the items are sortable by default. Maybe I can change the prop name to be clear and update the types as needed?